package.json

```diff
@@ -1,337 +1,71 @@
-      "endevorExplorerContainer": [
-        {
-          "id": "endevorExplorer",
-          "name": "Explorer for Endevor"
-        }
-      ]
-    }
-  },
   "scripts": {
-    "vscode:prepublish": "npm run compile",
-    "compile": "tsc -p ./",
-    "test:vscode": "ts-node ./test/runTest.ts",
-    "watch": "tsc -watch -p ./",
-    "test": "npm run test:no-vscode && npm run test:vscode",
-    "test:no-vscode": "jest --config jest.config.js --coverage",
-    "pretty": "prettier --write .",
-    "lint": "concurrently -n \"typescpt,_eslint_,prettier\" \"tsc --noemit \" \"eslint --ext .ts .\" \"prettier --check .\"",
-    "detect-cycles": "madge --circular --extensions ts ./"
+    "build": "yarn workspaces run build",
+    "lint": "yarn workspaces run lint",
+    "pretty": "yarn workspaces run pretty",
+    "test": "yarn workspaces run test",
+    "clean": "yarn workspaces run clean && rimraf node_modules"
+  },
+  "dependencies": {
+    "fp-ts": "^2.8.5",
+    "io-ts": "^2.2.12",
+    "promise-pool-tool": "^1.3.3"
   },
   "devDependencies": {
+    "@types/chai": "^4.2.14",
+    "@types/chai-as-promised": "^7.1.3",
+    "@types/chai-spies": "^1.0.2",
+    "@types/cheerio": "^0.22.22",
     "@types/glob": "^7.1.3",
-    "@types/jest": "^26.0.7",
+    "@types/jest": "^26.0.15",
+    "@types/js-yaml": "^3.12.6",
     "@types/mocha": "^8.0.3",
     "@types/node": "^14.14.5",
-    "@types/vscode": "^1.32.0",
-    "@typescript-eslint/eslint-plugin": "^4.7.0",
-    "@typescript-eslint/parser": "^4.7.0",
+    "@types/vscode": "^1.50.0",
+    "@typescript-eslint/eslint-plugin": "^4.6.0",
+    "@typescript-eslint/parser": "^4.6.0",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "chai-spies": "^1.0.0",
+    "cheerio": "^1.0.0-rc.3",
     "concurrently": "^5.3.0",
-    "eslint": "^7.13.0",
-    "eslint-config-prettier": "^6.15.0",
+    "eslint": "^7.12.1",
+    "eslint-config-prettier": "^6.14.0",
     "eslint-plugin-jest": "^24.1.0",
     "eslint-plugin-prettier": "^3.1.4",
+    "fast-check": "^2.9.0",
     "glob": "^7.1.6",
-    "jest": "^26.4.2",
-    "jest-html-reporter": "^3.2.0",
-    "madge": "^3.12.0",
+    "jest": "^26.6.1",
+    "jest-html-reporter": "^3.3.0",
+    "js-yaml": "^3.14.1",
+    "madge": "^3.11.0",
+    "mkdirp": "^1.0.4",
     "mocha": "^8.2.0",
-    "prettier": "2.1.2",
-    "ts-jest": "^26.4.0",
+    "mockttp": "^1.0.3",
+    "prettier": "^2.1.2",
+    "rimraf": "^3.0.2",
+    "sinon": "^9.2.0",
+    "ts-jest": "^26.4.3",
+    "ts-loader": "^8.0.7",
     "ts-node": "^9.0.0",
-    "typescript": "^3.5.3",
-    "vscode-test": "^1.4.1"
-  },
-  "dependencies": {
-    "@broadcom/endevor-for-zowe-cli": "^5.7.2",
-    "@zowe/imperative": "^4.7.4"
+    "typescript": "^4.1.3",
+    "vscode-test": "^1.4.1",
+    "webpack": "^4.44.2",
+    "webpack-cli": "^4.1.0",
+    "@local/eslint-config-local": "1.0.0",
+    "@types/selenium-webdriver": "^3.0.4",
+    "selenium-webdriver": "^3.6.0",
+    "testcontainers": "^7.4.0"
```

Comment:
do we need `testcontainers` dependency?

Reply:
Yep, we don't need it now. I deleted it, thank you!